src/hacking/building-and-running-on-wsl.md

```diff
@@ -0,0 +1,41 @@
+# Building and Running on WSL
+
+## Building
+
+Servo can be built on WSL as if you are building on any other Linux distribution.
+
+1. Setup WSL v2.
+2. Setup the environment depending on the WSL distribution that you are using (e.g. Ubuntu, OpenSuse, etc)
+3. Build
+
+## Running
+
+WSL v2 has the corresponding adaptors to display wayland and x11 applications, though it may not always work OOB with servo.
```

Comment:
Perhaps we should link the [help article for running GUI apps under WSL](https://learn.microsoft.com/en-us/windows/wsl/tutorials/gui-apps) and mention that Windows 11 or a somewhat recent windows 10 version is required.